src/main/java/org/symphonyoss/symphony/messageml/elements/Option.java

```diff
@@ -0,0 +1,50 @@
+package org.symphonyoss.symphony.messageml.elements;
```

Comment:
Reorganize your methods, as per Java standards, the order on a file is static above non-static methods, and public -> protected -> private in this order (of appearance) as well.